src/components/interview.ts

```diff
@@ -36,41 +52,72 @@ export async function handleInterview(message: Discord.Message, args: string[],
 }
 
 async function help(message: Discord.Message): Promise<void> {
+  //should be deprecated as soon as proper help handler is built.
   const response =
     "I can't seem to recognize your command; the commands I know for interviewers are: \n" +
-    ' ```.interviewer signup [calendly/xai link] ``` \n' +
-    ' ```.interviewer list ```';
+    ' ```.interviewer signup [calendly/xai link] \n//signs up or update your calendar link```' +
+    ' ```.interviewer list (domain) \n//shows list of available interviewers, with optional domain filter```' +
+    ' ```.interviewer domain [domain] \n//adds or removes you from a domain``` ';
   await message.channel.send(response);
 }
 
 async function addInterviewer(message: Discord.Message, args: string[]): Promise<void> {
   const db = await openDB();
   const id = message.author.id;
+
+  //checks for existence of link arg
   const link = args.shift();
```

Comment:
this takes the rightmost arg, should we maybe check the number of args supplied for better user input validation? again with Commando or slash commands we wouldn't have to worry about this.